### DISCUSSION TITLE
Consulo won't open projects in Linux.

### DISCUSSION THREAD

**Discussion by Hellvol** [consulo/consulo#467] [Q&A] [answered]
Hello. I've installed Consulo in my home folder. It starts, I can download the plugins and etc, and starts as usual. Then, when I try to open a project, this happens:
![image](https://user-images.githubusercontent.com/62046614/125204457-d34af500-e253-11eb-962e-ad94645838fb.png)

I can't open this locked folder, the root folder, and so, none of the folders or files in my machine.

Setting a project file to open as Consulo don't work either.

Any suggestions?

**Comment by VISTALL** [comment, marked as answer]
Hello. Can you attach log from welcome screen Configure -> Show log In ...

Thanks

**Reply by Hellvol** [reply]
[consulo.log](https://github.com/consulo/consulo/files/6797357/consulo.log)
Here it is.

**Reply by VISTALL** [reply]
What type of project you want to open?

**Reply by Hellvol** [reply]
Well I have the Unity and .Net modules installed. But I can't open any type. I can't browse my files to select any type of project, neither Java, neither php files, or anything.

**Reply by VISTALL** [reply]
It's MSBuild plugin issue. Disable it. I will publish new release builds at next week

**Reply by Hellvol** [reply]
It worked. Thank you!
Just a small question: I still need the Consulo Unity plugin to be in my Unity project to debug it?

**Reply by VISTALL** [reply]
Nope.
